docs/pages/Configuration.md

```diff
@@ -750,6 +751,15 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- Remove bundle option [`usePredefineCalls`](#properties) 
```

Comment:
```suggestion
- Remove bundle option [`usePredefineCalls`](#properties). UI5 CLI v4.0.0 and above will always use predefine calls in bundles, making this option obsolete.
```

Reply:
Just to give some context on why it has been removed and what the behvavior looks like now.